### PR TITLE
Update convert-user-mailbox-to-shared-mailbox.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated steps for new exchange admin center

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6140

File: microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 1]
```diff
@@ -58,6 +58,24 @@ For everything else you need to know about shared mailboxes, see [About shared m
 > [!NOTE]
 > Shared mailboxes don’t require a separate license. However, if you want to enable In-Place Archive or put an In-Place Hold or a Litigation Hold on a shared mailbox, you must assign an Exchange Online Plan 1 with Exchange Online Archiving or Exchange Online Plan 2 license to the mailbox.
 
+## Use the New Exchange admin center to convert a mailbox
+
+1. Go to the <a href="https://admin.exchange.microsoft.com/#/homepage" target="_blank"> Exchange admin center</a>.
+
+2. Select **Recipients** \> **Mailboxes**.
+
+3. Select the user mailbox. In the **Mailbox** tab, Click on **Convert to shared mailbox** under **More Actions**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md]:
```suggestion
3. Select the user mailbox. In the **Mailbox** tab, select **Convert to shared mailbox** under **More Actions**.
```

**Review by JohanFreelancer9 — APPROVED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**